packages/rum/src/domain/record/serializationUtils.spec.ts

```diff
@@ -95,3 +101,89 @@ describe('getElementInputValue', () => {
     })
   })
 })
+
+describe('Build stylesheet absolute url', () => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏since it is the test subject and it is what is done for other suites in his file
```suggestion
describe('switchToAbsoluteUrl', () => {
```